bgpd/bgp_evpn_vty.c

```diff
@@ -4131,14 +4135,18 @@ DEFUN(show_bgp_l2vpn_evpn_route,
 	/* get the type */
 	if (argv_find(argv, argc, "type", &type_idx)) {
 		/* Specific type is requested */
-		if (strncmp(argv[type_idx + 1]->arg, "ma", 2) == 0)
-			type = BGP_EVPN_MAC_IP_ROUTE;
-		else if (strncmp(argv[type_idx + 1]->arg, "mu", 2) == 0)
-			type = BGP_EVPN_IMET_ROUTE;
-		else if (strncmp(argv[type_idx + 1]->arg, "e", 1) == 0)
-			type = BGP_EVPN_ES_ROUTE;
-		else if (strncmp(argv[type_idx + 1]->arg, "p", 1) == 0)
-			type = BGP_EVPN_IP_PREFIX_ROUTE;
+                if ((strncmp(argv[type_idx + 1]->arg, "ma", 2) == 0) ||
```

Comment:
is this formatted by clang? I got different results in my editor :)

Reply:
This is wrong.